src/pyramid/config/routes.py

```diff
@@ -139,6 +140,26 @@ def add_route(
 
           .. versionadded:: 1.1
 
+        inherit_slash
+
+          This argument can only be used when the ``pattern`` is an empty
+          string (``''``). By default, the composed route pattern will always
+          includes a trailing slash, but this argument provides a way to
+          opt-out if both, you and the integrator (the developer setting the
+          :term:`route prefix`), agree that the pattern should not contain
+          a trailing slash. For example:
```

Comment:
I have a hard time parsing this paragraph so I simplified it. I assume that I am both "you" and "the integrator", but maybe that's deliberate. I also made some grammar fixes.

```
... include a trailing slash.
However this argument provides a way to opt out of the default and instead make the pattern not contain a trailing slash.
For example:
```

Reply:
I tried to improve this. Your proposed changes are not correct, opting into inheritance means there are two sides and I have to delineate them.